wordpress-mobile/WordPress-iOS
# InviteLinks: Adds a core data model for storing InviteLinks.

Refs #15356 

This PR adds an `InviteLinks` model to core data.  This will be used to store generated group invite links for p2 sites.  Changes include:
- A new model revision including the InviteLinks entity.  All properties and relations are non-optional.
- New NSModelObject subclass for InviteLinks
- Updated the Blog entity and class to have an optional one to many relation to InviteLinks.
- Updated MIGRATION.md

There are no other visual or functional changes with this PR.  It's just getting the model in place.

To test:
- Prep by ensuring your logged into the app.  Then switch to this branch.
- Build and run the app.  Confirm you were not logged out.
- Confirm that version 113 is the currently selected model revision.

@ScoutHarris would you kindly? 

PR submission checklist:
